build-system/tasks/test-report-upload.js

```diff
@@ -62,25 +63,25 @@ async function getReport(testType) {
  * @return {Object.<string,Object>} Object containing the build, job, and test results.
  */
 function addJobAndBuildInfo(testType, reportJson) {
-  const buildNumber = travisBuildNumber();
-  const commitSha = travisCommitSha();
-  const jobNumber = travisJobNumber();
+  const buildNumber = ciBuildNumber();
+  const commitSha = ciCommitSha();
```

Comment:
Since this runs on PRs, should this be `ciPullRequestSha()`?

Reply:
Side note: it's not easy to tell which to use, `ciPullRequestSha()` or `ciCommitSha()` unless you already knew how this file was being called. Maybe worth combining the 2 methods.

Reply:
Test case reporting is only done for push builds (in order to track flakiness on `master`), so `ciCommitSha()` is the correct choice here.

Re: combining the two methods, there are a handful of use cases that care about the difference (e.g. `git.js`). Also, services like Travis explicitly store the state in separate environment variables. Finally, I wanted to minimize changes to behavior / logic in this refactor, so I think leaving them as separate is the way to go. SGTY?

Reply:
Ah I mistook this for the GitHub PR checks. SG.